src/app/server/Mdns.cpp

```diff
@@ -51,7 +51,8 @@ NodeId GetCurrentNodeId()
     // mdns advertises a single node id as parameter.
 
     // Search for one admin pariing and use its node id.
-    for (auto pairing = GetGlobalAdminPairingTable().cbegin(); pairing != GetGlobalAdminPairingTable().cend(); pairing++)
+    auto pairing = GetGlobalAdminPairingTable().cbegin();
+    if (pairing != GetGlobalAdminPairingTable().cend())
```

Comment:
this would only check the first entry - the admin table in theory could have many entires.